data/json/mapgen/nested/city_npc_nested_spawns.json

```diff
@@ -45,18 +45,18 @@
     "object": {
       "mapgensize": [ 10, 10 ],
       "rows": [
-        "__._______",
-        "_L.7______",
-        "_L.7______",
-        "_L.7______",
-        "_L.7______",
-        "_L.7______",
-        "_L.7______",
-        "_L..mtb.__",
-        "_L.....___",
-        "_LLLLLL.__"
+        "  .       ",
```

Comment:
Is this planned or accidential? Seems there's `_` on the right and ` ` on the left.

Reply:
Yes, that was intentional.  When using `_`, it must have some mapgen function defined for it, so I had to swap it for ` ` which can have nothing defined for it.